AdobeStockClient/Model/Client.php

```diff
@@ -159,11 +159,13 @@ public function search(SearchCriteriaInterface $searchCriteria): SearchResultInt
                 $items[] = $this->convertStockFileToDocument($file);
             }
             $totalCount = $response->getNbResults();
-        } catch (Exception $e) {
-            if (strpos($e->getMessage(), 'Api Key is invalid') !== false) {
-                throw new AuthenticationException(__($e->getMessage()), $e, $e->getCode());
+        } catch (Exception $exception) {
```

Comment:
Because of Adobe Stock API specifics, it was decided to intentionally return the empty result instead of exception if the search fails for reasons other than incorrect credentials.